Core/MIPS/MIPSVFPUUtils.cpp

```diff
@@ -29,6 +29,13 @@
 #define V(i)   (currentMIPS->v[voffset[i]])
 #define VI(i)  (currentMIPS->vi[voffset[i]])
 
+// Flushes the angle to 0 if exponent smaller than this in vfpu_sin/vfpu_cos/vfpu_sincos.
+// Was measured to be around 0x68, but GTA on Mac is somehow super sensitive
```

Comment:
Well, it's not "about zero", the PSP doesn't return any non-zero value with an exponent below 0x68 (based on an exhaustive analysis of all possible input values that don't wrap.)  I'm not arguing with this change, though, as clearly macOS/iOS are returning values with lower exponents than that for values the PSP returns an exponent >= 0x68.

-[Unknown]